bin/c-db-import.js

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const program = require('commander');
+const { spawn } = require('child_process');
+const { loadConfig, reportError } = require('./lib/c');
+
+// The basic program, which uses sub-commands.
+program
+    .arguments('[env]')
+    .parse(process.argv);
+
+if (!program.args.length) {
+    return reportError(new Error('Please provide an environment'), program);
+}
+
+const [env] = program.args;
+
+if (env.toLowerCase() === 'local') {
+    return reportError(new Error('You cannot download the local database'), program);
+}
+
+Promise.all([
+    loadConfig(`mongodb.${env}`),
```

Comment:
@allanchau, I think it would be better to just execute `loadConfig('mongodb')` once, and then grab the environments yourself. More performant.